packages/nimble-components/src/breadcrumb-item/styles.ts

```diff
@@ -1,81 +1,101 @@
 import { css } from '@microsoft/fast-element';
 import { display } from '../utilities/style/display';
 import {
-    bodyFontColor,
     borderHoverColor,
-    borderWidth,
     controlHeight,
     iconSize,
+    linkActiveFontColor,
+    linkDisabledFontColor,
     linkFont,
+    linkFontColor,
+    mediumPadding,
     placeholderFontColor
 } from '../theme-provider/design-tokens';
 import { focusVisible } from '../utilities/style/focus';
 
 export const styles = css`
-    ${display('inline-flex')}
+    @layer base {
+        ${display('inline-flex')}
 
-    :host {
-        height: ${controlHeight};
-        padding-left: calc(4px - ${borderWidth});
+        :host {
+            height: ${controlHeight};
 
-        ${
-            /* When href removed the .control element is also removed
-             so this becomes the fallback color for the slot */ ''
+            ${
+                /* When href removed the .control element is also removed
+                so this becomes the fallback color for the slot */ ''
+            }
+            color: ${linkDisabledFontColor};
+            font: ${linkFont};
         }
-        color: ${bodyFontColor};
-        font: ${linkFont};
-    }
 
-    .listitem {
-        display: flex;
-        align-items: center;
-    }
+        .listitem {
+            display: flex;
+            align-items: center;
+        }
 
-    .control {
-        color: var(--ni-private-breadcrumb-link-font-color);
-        display: flex;
-        align-items: center;
-        justify-content: center;
-        border: ${borderWidth} solid transparent;
-        padding-right: calc(4px - ${borderWidth});
-        text-decoration: none;
-    }
+        .control {
+            color: ${linkFontColor};
+            display: flex;
+            align-items: center;
+            justify-content: center;
+            text-decoration: none;
+        }
 
-    .control:hover {
-        text-decoration: underline;
-    }
+        [part='start'] {
+            display: none;
+        }
 
-    .control${focusVisible} {
-        border: ${borderWidth} solid ${borderHoverColor};
-        outline: 2px solid ${borderHoverColor};
-        outline-offset: 1px;
-    }
+        .content {
+            pointer-events: none;
+        }
 
-    .control:active {
-        color: var(--ni-private-breadcrumb-link-active-font-color);
-        text-decoration: none;
-    }
+        [part='end'] {
+            display: none;
+        }
+
+        .separator {
+            padding-left: ${mediumPadding};
+            padding-right: ${mediumPadding};
+        }
```

Comment:
I'm not sure why we didn't go this route initially, but this seems way simpler and avoids the whole issue with styling the default slot, as well as the need to remove item padding via the parent breadcrumb's styles.